code/style.css

```diff
@@ -1,5 +1,156 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+img {
+  font-size: 14px;
+}
 body {
-  background-color: lightblue;
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+  margin: 100px 0px;
+  background-color: rgb(252, 253, 254);
+}
+h1 {
```

Comment:
Some space between the elements would be nice for better readability. Also - structuring your CSS after how the HTML is ordered is a good idea to be able to follow easier.

```
body {
  background-color: lightblue;
  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
  margin: 100px 0px;
  background-color: rgb(252, 253, 254);
}

h1 {
```